<p>I'm having a really frustrating problem using numpy's inverse fast fourier transform function. I know the fft function works well based on my other results.   Error seems to be introduced after calling ifft. The following should be printing zeros for example:</p>  <pre><code>temp = Eta[50:55]  print(temp)  print(temp-np.fft.fft(np.fft.ifft(temp))) </code></pre>  <p>Output:</p>  <pre><code>[ -4.70429130e+13 -3.15161484e+12j -2.45515846e+13 +5.43230842e+12j -2.96326088e+13 -4.55029496e+12j 2.99158889e+13 -3.00718375e+13j -3.87978563e+13 +9.98287428e+12j]  [ 0.00781250+0.00390625j -0.02734375+0.01757812j 0.05078125-0.02441406j 0.01171875-0.01171875j -0.01562500+0.015625j ] </code></pre>  <p>Please help!</p>